src/reducers/scrollStateHelper.js

```diff
@@ -32,27 +34,37 @@ import verticalHeightsSelector from 'verticalHeights';
  */
 function computeRenderedRows(state, scrollAnchor) {
   const newState = Object.assign({}, state);
-  const rowRange = calculateRenderedRowRange(newState, scrollAnchor);
-  computeRenderedRowOffsets(newState, rowRange);
+  let rowRange = calculateRenderedRowRange(newState, scrollAnchor);
 
   let {
-    firstRowIndex,
-    firstRowOffset,
-    rowHeights,
     rowsCount,
     scrollContentHeight,
-    scrollY,
   } = newState;
 
   const { bodyHeight } = verticalHeightsSelector(newState);
   const maxScrollY = scrollContentHeight - bodyHeight;
 
+  // NOTE (jordan) This handles #115 where resizing the viewport may
```

Comment:
Pulled this up out of calculateRenderedRowRange since in calculateRenderedRowRange we don't yet know an accurate body height.

If I left this in, we might end up with an edge case where we would always force you to be at the bottom of a table which was just larger than the viewport when the scrollbar was visible.